From 311d4759340f2af1e1e157d571802e4367e0a46b Mon Sep 17 00:00:00 2001
From: Yi Zhao <yi.zhao@windriver.com>
Date: Mon, 2 Aug 2021 09:38:39 +0800
Subject: [PATCH] fc/usermanage: update file context for chfn/chsh

The util-linux has provided chfn and chsh since oe-core commit
804c6b5bd3d398d5ea2a45d6bcc23c76e328ea3f. Update the file context for
them.

Upstream-Status: Inappropriate [embedded specific]

Signed-off-by: Yi Zhao <yi.zhao@windriver.com>
---
 policy/modules/admin/usermanage.fc | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/policy/modules/admin/usermanage.fc b/policy/modules/admin/usermanage.fc
index 6a051f8a5..bf1ff09ab 100644
--- a/policy/modules/admin/usermanage.fc
+++ b/policy/modules/admin/usermanage.fc
@@ -5,8 +5,10 @@ ifdef(`distro_debian',`
 /usr/bin/chage		--	gen_context(system_u:object_r:passwd_exec_t,s0)
 /usr/bin/chfn		--	gen_context(system_u:object_r:chfn_exec_t,s0)
 /usr/bin/chfn\.shadow		--	gen_context(system_u:object_r:chfn_exec_t,s0)
+/usr/bin/chfn\.util-linux		--	gen_context(system_u:object_r:chfn_exec_t,s0)
 /usr/bin/chsh		--	gen_context(system_u:object_r:chfn_exec_t,s0)
 /usr/bin/chsh\.shadow		--	gen_context(system_u:object_r:chfn_exec_t,s0)
+/usr/bin/chsh\.util-linux		--	gen_context(system_u:object_r:chfn_exec_t,s0)
 /usr/bin/crack_[a-z]*	--	gen_context(system_u:object_r:crack_exec_t,s0)
 /usr/bin/cracklib-[a-z]* --	gen_context(system_u:object_r:crack_exec_t,s0)
 /usr/bin/gpasswd	--	gen_context(system_u:object_r:groupadd_exec_t,s0)
-- 
2.17.1

